sdk/cosmos/azure-cosmos-spark_3-1_2-12/Samples/Python/NYC-Taxi-Data/01_Ingestion.ipynb

```diff
@@ -0,0 +1,271 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "inputWidgets": {},
+     "nuid": "b08b1d81-028c-4dba-ac24-b23ceebc9e38",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "**Secrets**\n",
+    "\n",
+    "The secrets below  like the Cosmos account key are retrieved from a secret scope. If you don't have defined a secret scope for a Cosmos Account you want to use when going through this sample you can find the instructions on how to create one here:\n",
+    "- Here you can [Create a new secret scope](./#secrets/createScope) for the current Databricks workspace\n",
```

Comment:
`/#secrets/createScope` seems to be invalid link.

is it meant to be this one?
https://docs.microsoft.com/en-us/azure/databricks/security/secrets/secret-scopes

Reply:
No - it is intentional - the relative link will point you to the Uri within your current Databricks workspace to create a new security scope